package.json

```diff
@@ -136,6 +136,7 @@
     "rollup-plugin-replace": "2.2.0",
     "rollup-plugin-uglify": "6.0.4",
     "scriptjs": "2.5.9",
+    "search-insights": "https://pkg.csb.dev/algolia/search-insights.js/commit/0b3c26c5/search-insights",
```

Comment:
The library is not released, so we have this temporary for now. We shouldn't merge this PR yet.